lib/active_model/serializer.rb

```diff
@@ -164,6 +164,14 @@ def json_key
       end
     end
 
+    def id
+      object.id if object
```

Comment:
It's converted to a string in the adapter (https://github.com/rails-api/active_model_serializers/pull/853/files#diff-ae1e38e96a8bf28ae37918a65c448c73R89)